build/jshint/xfail.list

```diff
@@ -210,7 +210,6 @@ apps/system/test/unit/mock_sheets_transition.js
 apps/system/test/unit/mock_stack_manager.js
 apps/system/test/unit/mock_statusbar.js
 apps/system/test/unit/mock_template.js
-apps/system/test/unit/mock_trusted_ui_manager.js
```

Comment:
This file seems to be unused, so it doesn't make sense to keep it in xfail.list.